code/server/main.js

```diff
@@ -49,19 +44,6 @@ function processOptions(args) {
             case options[1]:
                 db.clearDatabase(args.env);
                 break;
-            case options[2]:
-                api.injectUser(stubUser);
-                console.log("RUNNING WITH USER DB STUBBED");
-                break;
-            case options[3]:
-                api.injectLikes(stubLikes);
-                console.log("RUNNING WITH LIKES TABLE STUBBED");
-                break;
-            case options[4]:
-                api.injectLikes(stubUser);
-                api.injectLikes(stubLikes);
-                console.log("EVERYTHING IS A STUB!");
-                break;
```

Comment:
I'll add a function in the api file to handle that. We can run the function from our testing files to inject the stub into the DAOs.